src/main/java/makamys/neodymium/mixin/MixinWorldRenderer.java

```diff
@@ -150,6 +192,10 @@ private void notifyIfDrawnStatusChanged() {
     
     @Override
     public boolean isDrawn() {
-        return isInFrustum && (!skipRenderPass[0] || !skipRenderPass[1]);
+        if (!(skipRenderPass[0] && skipRenderPass[1]))
```

Comment:
Frustum checks are now ignored in all cases, I doubt that was the intention.
```suggestion
        return (isInFrustum || Compat.isOptiFineShadersEnabled()) && !(skipRenderPass[0] && skipRenderPass[1]);
```